pkgs/applications/networking/cluster/fluxcd/default.nix

```
@@ -1,25 +1,26 @@
 { lib, buildGoModule, fetchFromGitHub, fetchzip, installShellFiles }:
 
 let
-  version = "0.17.2";
+  pkgVersion = "0.17.2";
```

```suggestion
  version = "0.17.2";
```
Renaming the others should be fine I guess.

---

Done